### PR TITLE
Travis fixup

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

**Comment by wronglink** [comment]
looks like pytest30 should be added to tox environment list

File: README.rst [hunk 1]
```diff
@@ -258,5 +258,5 @@ where the configuration file was found.
 Issue and Bug Tracker
 ------------------------
 
-Please use the `pytest issue tracker <https://github.com/pytest-dev/pytest/issues>`_
+Please use the `pytest issue tracker <https://github.com/pytest-dev/pytest-xdisr/issues>`_
```
Comment (nicoddemus) [inline README.rst]: Typo

File: .travis.yml [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
-  - TESTENV=py34-pytest29
-  - TESTENV=py35-pytest27
-  - TESTENV=py35-pytest28
-  - TESTENV=py35-pytest29
-  - TESTENV=py27-pytest28-pexpect
-  - TESTENV=py35-pytest28-pexpect
-  - TESTENV=flakes
-  - TESTENV=readme
-script: tox --recreate -e $TESTENV
+  - TOXENV=py-pytest27
```
Comment (nicoddemus) [inline .travis.yml]: Why using `py-pytest27` instead of simply `pytest27`?

File: .travis.yml [hunk 3]
```diff
@@ -1,37 +1,40 @@
...
+- '3.3'
+- '3.4'
 - '3.5'
+- nightly
+
+cache: pip
 install: pip install -U tox setuptools_scm
 env:
   matrix:
-  - TESTENV=py26-pytest26
```
Comment (nicoddemus) [inline .travis.yml]: I'm curious, why did you change how the environment was declared? TBH I prefer the old way, because it is more explicit and can easily be updated by using the output from `tox -l`...

File: .travis.yml (hunk 2 above)
Comment (RonnyPfannschmidt) [inline .travis.yml]: morbid habit wrt factors, thanks for noticing

File: .travis.yml (hunk 3 above)
Comment (RonnyPfannschmidt) [inline .travis.yml]: it gives more correct results from travis
i#d like to see a tox plugin to work directly on top of that

**Comment by RonnyPfannschmidt** [comment]
the remaining test issues will be fixed after i extract a version picking library from a work project today

File: .travis.yml (hunk 3 above)
Comment (nicoddemus) [inline .travis.yml]: > it gives more correct results from travis

Could you elaborate? I'm curious because most of my projects use this scheme so I'm interested if there's anything wrong with that approach.

Or do you mean visually the jobs on Travis better reflect the Python version running? If that's the case, there's [tox-travis](https://pypi.python.org/pypi/tox-travis), but never used it for non-trivial tox configurations such as what we use here.

**Comment by RonnyPfannschmidt** [comment]
closing after merging the one by @nicoddemus (all-tho it works slightly different wrt travis integration)
